.github/workflows/rolling-abi-compatibility.yml

```diff
@@ -1,8 +1,9 @@
 name: ABI Compatibility Check
 on:
-  pull_request:
-    branches:
-      - master
+  workflow_dispatch:
```

Comment:
Hi @destogl, just wished to check this as I look through the pr, this workflow is only expected to be run manually for now and not on push?